pycalphad/tests/test_calculate.py

```diff
@@ -240,3 +240,15 @@ def test_charged_infeasible_minimum_norm():
     # Check that the point sample didn't get 'stuck' in part of the space
     assert np.any(np.logical_and(output[:, 1] > 0.05, output[:, 1] < 0.15))
     assert np.any(np.logical_and(output[:, 1] > 0.25, output[:, 1] < 0.35))
+
+
+@select_database("Al-Fe_sundman2009.tdb")
+def test_BCC_phase_with_symmetry_option_B(load_database):
```

Comment:
I don't have a database like this one that has both models or easy access to another software for a test. I'm not opposed to the test.

Strictly, it shouldn't be necessary because this test shows that option B is correct and we have orthogonal tests that the symmetry groups of option F are generated correctly.

I'll note that I did an eye test on the Al-Ni phase diagram from `mpea-02b.tdb` database and it is clearly incorrect when plotted using `develop` and correct when plotted using this branch.